tests/integration-tests/README.md

```diff
@@ -976,3 +976,30 @@ Scheduling:
         SubnetIds:
           - {{ public_subnet_id }}
 ```
+## Collect system report
+
+In case of performance degradation detected in tests, to speed up the root cause analysis it could be useful to have a
+report on all the service, cron and information about the  . The function create
```

Comment:
cluster?

Reply:
Hi @tschuett,
thank you for your feedback, the readme it is still work in progress; when the PR will be ready I would appreciated a feedback from you.